qiskit/circuit/library/n_local/efficient_su2.py

```diff
@@ -114,10 +114,13 @@ def efficient_su2(
     if su2_gates is None:
         su2_gates = ["ry", "rz"]
 
+    # Set entanglement_blocks to None when num_qubits == 1
+    entanglement_blocks = ['cx'] if num_qubits > 1 else []
```

Comment:
For this PR you should make sure to run the formatter (`black`) so the CI passes. I think here it will complain about the single quotes. You can run the formatter by executing `make black` in the root directory.